oeps/processes/oep-0062-proc-working-groups.rst

```diff
@@ -0,0 +1,191 @@
+
+OEP-62: Working Groups
+######################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-62 <oep-0062-proc-working-groups>`
+   * - Title
+     - Working Groups
+   * - Last Modified
+     - 2023-01-XX
+   * - Authors
+     - Kyle McCormick <kyle@tcril.org>
+   * - Arbiter
+     - <Arbiter's real name and email address>
+   * - Status
+     - Accepted
+   * - Type
+     - Process
+   * - Created
+     - 2022-12-16
+   * - Review Period
+     - 2023-01-XX - 2023-01-XX
+   * - Resolution
+     - `Pull Request <https://github.com/openedx/open-edx-proposals/pull/424>`_
+   * - References
+     - * `List of working groups <https://openedx.atlassian.net/wiki/spaces/COMM/pages/46793351/Open+edX+Working+Groups>`_
+       * `Working group calendar <https://calendar.google.com/calendar/u/0?cid=Y192ODZzaHJuZWdzaHNxZ3A0Zmoyazk0dTdiY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t>`_
+
+Abstract
+********
+
+Working groups are voluntary associations of Open edX community members formed around some aspect of the project. These groups are organically formed and dissolved by consensus amongst those interested. While they must follow a few basic community guidelines, working groups have autonomy to decide their own structure and operating procedures.
+
+Even before this OEP was codified, working groups played a vital role in sustaining and advancing the Open edX platform. In this OEP we aim to document their successful practices for the benefit of current and future Open edX community members.
+
+Background & Motivation
+***********************
+
+The first community-lead Open edX working groups were the *Transifex Working Group* and the *Build-Test-Release Working Group*. The former group took responsibility for creating and curating translations of strings on the Open edX platform; the latter took responsility for the platform's named releases. These groups were formed by community members who saw a need in the project and took the initiative to organize and solve that need.
+
+Since then, several more working groups have organized themselves using a similar model. The groups operate autonomously, but have largely standardized around a set of best practices to foster an inclusive and productive environment. Many of the groups play important roles in the project, such as organizing community events, developing standard user interface components, and driving project strategy. As a whole, working groups provide a space for Open edX contributors from different organizations and geographic regions to collaborate and form working relationships.
+
+Given the importance of working groups to the health of the Open edX project, it may surprise new Open edX contributors that litle formal documentation exists to establish the working group system. The only official guidance so far is found in the `Technical Oversight Committee (TOC) Charter`_:
```

Comment:
```suggestion
Given the importance of working groups to the health of the Open edX project, it may surprise new Open edX contributors that little formal documentation exists to establish the working group system. The only official guidance so far is found in the `Technical Oversight Committee (TOC) Charter`_:
```